src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Interop/HwndHost.cs

```diff
@@ -998,6 +997,10 @@ private void BuildOrReparentWindow()
                         // shutdown event after this HwndHost.
                         SystemResources.DelayHwndShutdown();
                     }
+                    else
+                    {
+                        Trace.WriteLineIf(hwnd == null, $"- Warning - Notification Window is null\n{new System.Diagnostics.StackTrace(true).ToString()}");
```

Comment:
I do not think we should check `hwnd` is null twice. Because only when hwnd is null, will it enter this branch.